src/pages/CustomerRequestOverduePayment/components/EmailPreview.tsx

```diff
@@ -0,0 +1,246 @@
+            )}
+        </Stack>
+        <Table
+          name="email-preview"
+          containerSize={{ default: 0 }}
+          rowSize={44}
+          data={invoices}
+          columns={[
+            {
+              key: 'number',
+              title: <Caption noWrap>{translate('text_6419c64eace749372fc72b3c')}</Caption>,
+              maxSpace: true,
+              content: ({ number }) => (
+                <Caption color="primary600" noWrap>
+                  {number}
+                </Caption>
+              ),
+            },
+            {
+              key: 'totalAmountCents',
+              textAlign: 'right',
+              title: <Caption noWrap>{translate('text_6419c64eace749372fc72b3e')}</Caption>,
+              content: (row) => (
+                <Caption color="textSecondary" noWrap>
+                  {intlFormatNumber(
+                    deserializeAmount(row.totalAmountCents, row.currency || currency),
+                    {
+                      currency: row.currency || currency,
+                      locale: documentLocale,
+                      currencyDisplay: 'narrowSymbol',
+                    },
+                  )}
+                </Caption>
+              ),
+            },
+          ]}
+        />
+
+        {organization?.email && (
+          <Box textAlign="center">
+            <Caption component="span">{translate('text_64188b3d9735d5007d712276')}</Caption>
+            <Caption component="span">{` `}</Caption>
+            <Caption component="span" color="primary600">
+              {organization?.email}
+            </Caption>
+          </Box>
+        )}
+      </Card>
+      <Stack direction="row" alignItems="center" gap={1} marginX="auto">
+        <Note color="grey500">{translate('text_6419c64eace749372fc72b03')}</Note>
+        <Logo height="12px" />
+      </Stack>
+    </Stack>
+  )
+}
+
+export const EmailPreviewSkeleton = () => {
+  return (
+    <Stack flexDirection="column" gap={8} maxWidth={600} marginX="auto">
+      <Header>
+        <Skeleton variant="connectorAvatar" size="medium" color="dark" />
+        <Skeleton variant="text" width={120} height={12} color="dark" />
+      </Header>
+      <Card $childSpacing={4}>
+        <Skeleton variant="text" width={104} height={12} color="dark" />
+        <Skeleton variant="text" width="100%" height={12} color="dark" />
+        <Skeleton variant="text" width={160} height={12} color="dark" />
+      </Card>
+      <Stack direction="row" alignItems="center" width="100%" justifyContent="center">
+        <Skeleton variant="text" width={120} height={12} color="dark" />
+      </Stack>
+    </Stack>
+  )
+}
+
+const Header = styled.div`
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  gap: ${theme.spacing(3)};
+  width: 100%;
+`
+
+const Text = styled(Typography)`
+  font-family: Helvetica, sans-serif;
+  font-size: 16px;
+  font-weight: 400;
+  line-height: 24px;
+`
+
+const CompanyName = styled(Text)`
```

Comment:
Why not using the Typography component here?

Reply:
`Text` is a custom component from `Typography` defined on line 212

Reply:
Sure, but what's the advantage of using Text over out internal Typography?

Reply:
Cause for the email preview we don't use the same font family...